src/chrome/content/rules/Tmblr.co.xml

```diff
@@ -1,21 +1,20 @@
 <!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://tmblr.co/ => https://www.tumbler.com/: (51, "SSL: certificate subject name 'www.tervis.com' does not match target host name 'www.tumbler.com'")
-	For other Tumblr coverage, see Tumbler.xml.
+	For other Tumblr coverage, see Tumblr.xml.
 
 
 	^: refused
 	www doesn't exist
 
 -->
-<ruleset name="Tmblr.co" default_off='failed ruleset test'>
+<ruleset name="Tmblr.co">
 
 	<target host="tmblr.co" />
 
+	<test url="http://tmblr.co/ZE5Fby1dP0xGe" />
 
 	<!--	Redirect keeps path and args:
 						-->
 	<rule from="^http://tmblr\.co/+"
```

Comment:
I agree the `+` is weird. It's harmless, so I decided not to make extraneous changes and complicate the diff. I can fix it later, but i don't have time right now.